man/zeroPad.Rd

```diff
@@ -22,6 +22,8 @@ pCode <- '10'
 correctPCode <- zeroPad(pCode,5)
 pCodes <- c('100','1000','0','12345','1565465465465465')
 correctPCodes <- zeroPad(pCodes,5)
+pCodeNA <- c(1,2,NA)
```

Comment:
@wdwatkins I didn't notice this in the code review on your pull request. You need to make sure that your .Rd files are getting rebuilt....I'll show you what I mean.